assignment.rb

```diff
@@ -0,0 +1,85 @@
+class Yarn
+  attr_accessor :weight, :num_of_plies, :is_machine_wash, :yardage, :quantity
+
+  def total_yds
+    quantity * yardage
+  end
+
+  def use(type = "scarf")
+    puts "I will use this yarn to make a #{type}."
+  end
+
+  def rating
+    puts "Yarn is awesome!"
+  end
+end
+
+class Animal_Fiber < Yarn
+  attr_accessor :is_organic, :is_next_to_skin
+end
+
+class Plant_Fiber < Yarn
+  attr_accessor :is_organic
```

Comment:
My thinking was that there are no organic manmade fibers (at least with yarn, as far as I know) so it would not apply. I suppose it could be in the superclass and always return false, but no one ever wonders if their acrylic yarn is organic.